tier3/checklist.md

```diff
@@ -0,0 +1,441 @@
+# DSAC OSPO Outbound Review Checklist
+## Tier 3: Public Repository
+### Instructions
+This is a review process to approve CMS-developed software to be released open source.
+If you would like your repository to be released, please complete the following steps.
+
+[Instructions](#instructions)
+
+[State the Benefit(s) of Open Sourcing the Project](#state-the-benefits-of-open-sourcing-the-project)
```

Comment:
Please rename to "State the Benefits of Open Sourcing the Project" and update the link accordingly